src/hyperlight_common/src/flatbuffer_wrappers/function_call.rs

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2024 The Hyperlight Authors.
+Copyright 2025 The Hyperlight Authors.
```

Comment:
Only add the header file to the files that are missing the header.  Don't update any of the files that already have it.

Reply:
you didn't get this right, don't touch any of the files with headers already present